HISTORY.md

```diff
@@ -1,6 +1,35 @@
 # Release History - `open-autonomy`
 
 
+# 0.5.0 (2022-12-05)
+
+Autonomy:
+- Introduces more flexible approach to defining the overrides for multiple agents in a service component
+- Deprecates the usage of `autonomy hash all` command, `autonomy packages lock` can be used to perform package dependency hash updates and checks
+- Updates the override serialisation mechanism to be consistent with the environment variable parser
+- Extends the `from-token` command to provide password for encrypted private keys at the runtime
+- Updates the process of overriding the safe contract address at the runtime to be more generalised
+- Fixes a bug that allowed the image build to continue even after a command run failed when building the image on `build-image` command
+ 
+Packages:
+- Fixes the `AbciApp` initialization to ensure that synchronized data is retrieved as an instance of the `synchronized_data_class` specified on the round.
+- Updates the `IPFSInteract` tool to
+  - Catch the broken connection exceptions
+  - Remove the correct path before downloading
+- Fixes the tendermint reset mechanism to 
+  - Avoid race conditions when performing a hard reset
+  - Not updated the initial height and genesis time when resetting
```

Comment:
Not update the initial height and genesis time when resetting for recovering agent <-> tendermint communication

Reply:
"when it is used as a recovery mechanism for broken agent <-> tendermint communication" - #1608
when there is no failure then these are updated.